Source/Charts/Utils/ChartUtils.swift

```diff
@@ -99,55 +99,65 @@ extension CGPoint
 
 open class ChartUtils
 {
-    private static var _defaultValueFormatter: IValueFormatter = ChartUtils.generateDefaultValueFormatter()
-    
-    open class func drawImage(
-        context: CGContext,
-        image: NSUIImage,
-        x: CGFloat,
-        y: CGFloat,
-        size: CGSize)
+    private static var _defaultValueFormatter: IValueFormatter = generateDefaultValueFormatter()
```

Comment:
is this from another PR? Or should in this PR

Reply:
This was just a minor change (ChartUtils.generateDefaultValueFormatter()` -> `generateDefaultValueFormatter()` because they're both static methods.) Nothing else. I can revert it if you want.